services/core/Darksky/darksky/data/name_mapping.csv

```diff
@@ -19,22 +19,22 @@ sunriseTime,,,
 sunsetTime,,,
 moonPhase,,,
 precipIntensityMax,,,
-temperatureHigh,,,
+temperatureHigh,,degC,degK
```

Comment:
This will not work with current code. The base weather test does not test this case either. Currently we assume that there would be valid values for standard point name if there is units.. we don't default to service point name if there is entry in csv with standard point name and units alone